ui.h

```diff
@@ -812,6 +812,29 @@ _UI_EXTERN int uiEntryReadOnly(uiEntry *e);
  */
 _UI_EXTERN void uiEntrySetReadOnly(uiEntry *e, int readonly);
 
+/**
+ * Returns the entry's placeholder.
+ *
+ * @param e uiEntry instance.
+ * @returns The placeholder text of the entry.\n
+ *          A `NUL` terminated UTF-8 string.\n
+ *          Caller is responsible for freeing the data with `uiFreeText()`.
+ * @memberof uiEntry
+ */
+_UI_EXTERN char *uiEntryPlaceholder(uiEntry *e);
+
+/**
+ * Sets text to be displayed in the entry when it is empty.
+ *
+ * @param e uiEntry instance.
+ * @param text Placeholder text.\n
+ *             A valid, `NUL` terminated UTF-8 string.\n
+ *             Data is copied internally. Ownership is not transferred.
+ * @warning Read-only entries can't not display the placeholder on Windows
```

Comment:
```suggestion
 * @warning Read only entries do not display the placeholder text on Windows.
```
Spelling and consistent punctuation.

And I confirmed as well that `EM_SETCUEBANNER` does not play nice with `ES_READONLY`.

Reply:
> Read-only -> Read only

I can see some comments that use "read-only" in the repo. (e.g. [windows/multilineentry.cpp](https://github.com/libui-ng/libui-ng/blob/257f2664b18df67778b68318125756ae691ce1e2/windows/multilineentry.cpp#L4))
But should I use "Read only" in the document?

> can't -> do

I thought I fixed it already but I seemed to revert it somehow...

Reply:
fixed at 3302ef9d668b23cbecfa6b37b46a7c9cd0e13d56